spec/spec_helper.coffee

```diff
@@ -1,9 +1,7 @@
-chai = require 'chai'
-sinon = require 'sinon'
-chai.use require 'sinon-chai'
-
-global.chai = chai
+global.chai = require 'chai'
 global.expect = chai.expect
-global.sinon = sinon
+global.sinon = require 'sinon'
+
+chai.use require 'sinon-chai'
 
 process.env.NODE_ENV = 'test'
```

Comment:
Yeah, this is fine. This was just the default helper that mycha gives you. I hadn't changed it from default.